### DISCUSSION TITLE
HSS Build fail for "no rule to make target"

### DISCUSSION THREAD

**Discussion by Mendandukka** [polarfire-soc/polarfire-soc-documentation#272] [HSS and U-Boot] [answered]
Hi!

I was trying to build HSS on RCHD-PF board. After the create .config file and selected this board, I have error in HSS build process. This error belong at attach. 

Thanks in advance for your help.

![GITHUB_SS](https://user-images.githubusercontent.com/55929243/210251627-d151bd4b-0bcd-41de-be9b-6f5c174929d5.png)

**Comment by TommyMurphyTM1234** [comment, marked as answer]
For some reason the `tool_versions.h` header file cannot be located when compiling `application/hart0/hss_init.c`.

* https://github.com/polarfire-soc/hart-software-services/blob/8af865f74810f216329606be7721da939239e68a/application/hart0/hss_init.c#L176

You probably need to double check the contents of your project and/or the include directives in the project settings or `Makefile`.
